packages/app/app/styles/views/reports.less

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright 2019, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+.navi-report {
+  &__breadcrumb,
+  &__header,
+  &__footer {
+    padding: 0 20px;
```

Comment:
Can we make this the same as the dashboard?
https://github.com/yahoo/navi/pull/410/files#diff-035417c76de71435bce60d8f3cb07391R12

Reply:
I intended to preserve the original padding while extending the gray background, but yeah I can make them more consistent